internal/controller/bindings/endpointbinding_poller.go

```diff
@@ -154,7 +154,7 @@ func (r *EndpointBindingPoller) createBinding(ctx context.Context, hashedName st
 	return nil
 }
 
-func (r *EndpointBindingPoller) updateBinding(ctx context.Context, binding *bindingsv1alpha1.EndpointBinding, apiEndpoint *EndpointBinding, urlBits *URLBits) error {
+func (r *EndpointBindingPoller) updateBinding(ctx context.Context, binding *bindingsv1alpha1.EndpointBinding, _ *EndpointBinding, urlBits *URLBits) error {
```

Comment:
Heads up we might conflict on this change

Reply:
I can drop it out that commit easily if needed.